integrationTests/vm/txsFee/multiShard/asyncESDT_test.go

```diff
@@ -136,7 +136,7 @@ func TestAsyncESDTTransferWithSCCallShouldWork(t *testing.T) {
 
 	utils.ProcessSCRResult(t, testContextFirstContract, intermediateTxs[1], vmcommon.Ok, nil)
 
-	require.Equal(t, big.NewInt(278300), testContextFirstContract.TxFeeHandler.GetAccumulatedFees())
+	require.Equal(t, big.NewInt(4936720), testContextFirstContract.TxFeeHandler.GetAccumulatedFees())
```

Comment:
There is quite a large change on the expected values.
how can we check that this value is correct?

Reply:
checked manually the code and we get the new values. It was a problem on builtInFunction costs.